easysid.cpp

```diff
@@ -26,7 +37,13 @@ EXPORT void start_capture(const char *fn)
         delete wav;
         wav = NULL;
     }
-    wav = makePcmOutputWav(fn);
+    if (pcm == NULL) {
+        if (not initAudioOutput()) {
+            // TODO: Error condition
+            return;
+        }
+    }
+    wav = makePcmOutputWav(fn, pcm->getSampleRate());
```

Comment:
Why does this rely on the sample rate of the pcm object? If you've already got a global g_sample_rate, then both the audio and the capture should use g_sample_rate in the same way. So just read from the global here.